themes/default-bootstrap/js/order-opc.js

```diff
@@ -924,7 +924,7 @@ function multishippingMode(it)
 						});
 					},
 					'afterLoad': function(){
-						$('.fancybox-inner .cart_quantity_input').typeWatch({ highlight: true, wait: 600, captureLength: 0, callback: function(val) { updateQty(val, false, this.el);} });
+						$('.fancybox-inner .cart_quantity_input').typeWatch({ highlight: true, wait: 600, captureLength: 1, callback: function(val) { updateQty(val, false, this.el);} });
```

Comment:
Hi,

With the new version of typewatch, ```this.el``` must be replaced by ```this```.

Best regards